### DISCUSSION TITLE
Can this library be used for encoder only models?

### DISCUSSION THREAD

**Discussion by elyxlz** [CarperAI/trlx#464] [Q&A] [answered]
I would like to use tune an encoder-only model (like T5) via RLHF and use its produced embeddings to calculate the reward.
Would this be possible with this library?
Thank you!

**Comment by maxreciprocate** [comment, marked as answer]
Hi! Unfortunately no, this library, at its current state, operates solely on discrete token level rewards. Although I recall that @shahbuland was doing something interesting with reinforcement on continuous action spaces, so he might give some pointers here

**Comment by elyxlz** [comment]
Thank you! I'll contact him
